Tests/AppTests/Extensions/HTTP/Responder/Responder+Authorization.swift

```diff
@@ -0,0 +1,68 @@
+import HTTP
+
+extension Responder {
+  public func unauthorizedTestResponse(
+    to method: HTTP.Method,
+    at path: String,
+    hostname: String = "0.0.0.0",
+    headers: [HTTP.HeaderKey : String] = [:],
+    body: BodyRepresentable? = nil,
+    file: StaticString = #file,
+    line: UInt = #line
+    ) throws -> HTTP.Response {
+    return try self.testResponse(
+      to: method,
+      at: path,
+      hostname: hostname,
+      headers: headers,
+      body: body,
+      file: file,
+      line: line
+    )
+  }
+  
+  public func appAuthorizedTestResponse(
+    to method: HTTP.Method,
+    at path: String,
+    hostname: String = "0.0.0.0",
+    headers: [HTTP.HeaderKey : String] = [:],
+    body: BodyRepresentable? = nil,
+    file: StaticString = #file,
+    line: UInt = #line
+    ) throws -> HTTP.Response {
+    var appHeaders = headers
+    appHeaders["client-token"] = "test"
```

Comment:
I'm wondering, what if we change our client-token from "test" to new value. I think, we will need to update this line accordingly. 

So maybe do something like this to take value from config here? 
```
let droplet = self as? Droplet
let token = droplet?.config["server", "client-token"]?.string ?? "test"
appHeaders["client-token"] = token 
```

Reply:
FIxed